### PR TITLE
Update syntax and improve perf, along with fix what will show up in H…

### PR DESCRIPTION
**Pull request by shainw**
…ostCustomEntity

Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Hunting Queries/AuditLogs/RareAuditActivityByApp.yaml [hunk 1]
```diff
@@ -19,35 +19,37 @@ query: |
   let current = 1d;
   let auditLookback = 14d;
   let AuditTrail = AuditLogs
```
Comment (ashwin-patil) [inline Hunting Queries/AuditLogs/RareAuditActivityByApp.yaml:21]: May want to add Global AllowList for Known Operations by Legit App occurring less frequent and classifying as rare.

Saw benign noise with below combination

- InitiatedByApp == "Microsoft Azure AD Group-Based Licensing"
- OperationName == "Change user license"

File: Hunting Queries/AuditLogs/RareAuditActivityByUser.yaml [hunk 2]
```diff
@@ -19,33 +19,34 @@ query: |
   let current = 1d;
   let auditLookback = 14d;
   let AuditTrail = AuditLogs
```
Comment (ashwin-patil) [inline Hunting Queries/AuditLogs/RareAuditActivityByUser.yaml:21]: Same as first about adding Global AllowList.

**Review by ashwin-patil — CHANGES_REQUESTED** [review]
Suggestion to add AllowList , rest looks good.

File: Hunting Queries/AuditLogs/RareAuditActivityByApp.yaml (hunk 1 above)
Comment (shainw) [reply Hunting Queries/AuditLogs/RareAuditActivityByApp.yaml:23]: done.

File: Hunting Queries/AuditLogs/RareAuditActivityByUser.yaml (hunk 2 above)
Comment (shainw) [reply Hunting Queries/AuditLogs/RareAuditActivityByUser.yaml:22]: Can only do the propertyName as there is no App information as this is done by a person.

**Review by ashwin-patil — APPROVED** [review]

**Pull request merged**